modules/document_repository/jsx/childTree.js

```diff
@@ -42,17 +70,21 @@ class ChildTree extends Component {
           style={trStyle}
         >
           <td colSpan="10">
-            <span
-              style={spanStyle}
+            <div
+              style={headerRow}
               key={index}
               className="pointer"
-              style={spanStyle}
             >
-              <i className="fa fa-folder"></i>
-              <span style={spanStyle}>
+              <i className="fa fa-folder" style={iconStyle}></i>
+              <div style={spanStyle} id="tooltip">
```

Comment:
`id`s need to be unique across the entire page. I think this component is used multiple times on the same page, so a hardcoded "tooltip" won't work.

Reply:
This PR does not work on my sandbox (no tooltip shown...)